content/administration/install/install.rst

```diff
@@ -505,50 +505,71 @@ create a new PostgreSQL user:
 
 .. code-block:: console
 
-    $ sudo apt install python3-dev libxml2-dev libxslt1-dev libldap2-dev libsasl2-dev \
-        libtiff5-dev libjpeg8-dev libopenjp2-7-dev zlib1g-dev libfreetype6-dev \
-        liblcms2-dev libwebp-dev libharfbuzz-dev libfribidi-dev libxcb1-dev libpq-dev
+   $ cd /CommunityPath
+   $ sed -n -e '/^Depends:/,/^Pre/ s/ python3-\(.*\),/python3-\1/p' debian/control | sudo xargs apt-get install -y
 
-Odoo dependencies are listed in the `requirements.txt` file located at the root of the Odoo
+The pip way
+~~~~~~~~~~~
+
+Alternatively, you can use pip to install python dependencies. As some of the packages need a
+compilation step, they require system libraries to be installed. On Debian/Ubuntu based systems, the
+following command should install these required libraries:
+
+.. code-block:: console
+
+   $ sudo apt install python3-pip libldap2-dev libpq-dev libsasl2-dev
+
+Odoo dependencies are listed in the :file:`requirements.txt` file located at the root of the Odoo
 community directory.
 
-.. tip:: It can be preferable to not mix python modules packages between different instances of Odoo
-         or with your system. You can use virtualenv_ to create isolated Python environments.
+.. note::
+   The python packages in :file:`requirements.txt` are based on their stable/LTS Debian/Ubuntu
+   corresponding version at the moment of the Odoo release.
+   e.g.: For Odoo 15.0, the `python3-babel` package version is 2.8.0 in Debian Bullseye and 2.6.0
+   in Ubuntu Focal. The lowest version is then chosen in the :file:`requirements.txt`.
 
-Navigate to the path of your Odoo Community installation (`CommunityPath`) and run **pip**
-on the requirements file:
+Navigate to the path of your Odoo Community installation (:file:`CommunityPath`) and run **pip** on
+the requirements file to install the requirements for the current user:
 
 .. code-block:: console
 
-    $ cd /CommunityPath
-    $ pip3 install setuptools wheel
-    $ pip3 install -r requirements.txt
+   $ cd /CommunityPath
+   $ pip install -r requirements.txt
 
-.. warning:: `wkhtmltopdf` is not installed through **pip** and must be installed manually in
-             version `0.12.5 <the wkhtmltopdf download page_>`_ for it to support headers and
-             footers. See our `wiki <https://github.com/odoo/odoo/wiki/Wkhtmltopdf>`_ for more
-             details on the various versions.
+.. tip::
+   It can be preferable to not mix python modules packages between different instances of Odoo or
+   with your system. You can use virtualenv_ to create isolated Python environments.
 
-For languages with right-to-left interface (such as Arabic or Hebrew), the package `rtlcss` is
-needed:
+Other Dependencies
+^^^^^^^^^^^^^^^^^^
 
-#. Download and install **nodejs** and **npm** with your package manager.
-#. Install `rtlcss`:
+- `wkhtmltopdf` must be installed manually in version `0.12.5 <the wkhtmltopdf download page_>`_ for
```

Comment:
@d-fence @AntoineVDV 

I'm putting this url https://wkhtmltopdf.org/downloads.html instead of the placeholder URL.
Or should it be  https://github.com/wkhtmltopdf/wkhtmltopdf/releases/0.12.5/   ?

Reply:
The later.

Reply:
Okay